src/install/config/memcached.yaml

```diff
@@ -0,0 +1,87 @@
+agentName: memcached
```

Comment:
I don't see `src/install/memcached/default-configuration.mdx` in this yaml but it exists in the folder.